astilectron.go

```diff
@@ -325,10 +325,13 @@ func (a *Astilectron) executeCmd(cmd *exec.Cmd) (err error) {
 // watchCmd watches the cmd execution
 func (a *Astilectron) watchCmd(cmd *exec.Cmd) {
 	// Wait
```

Comment:
I realize now that the content of this method needs to be wrapped in the following:

```go
a.worker.NewTask().Do(func() {
  // TODO Add content of method here
})
```

Indeed, this makes sure that the app waits for the execution of this logic before exiting. Otherwise, as this is executed in a goroutine, there's no guarantee it will be executed until the end. Therefore there's no guarantee your log message will be printed.

Could you add that?

Reply:
Sure